### DISCUSSION TITLE
Event listener doesn't contains partitions info while the query doesn't contains parition key as filter condition.

### DISCUSSION THREAD

**Discussion by awkwardd** [trinodb/trino#23992] [Q&A]
When use http event listener to get the partition info of query, I find that  if the query doesn't contains parition key as filter condition, the event doesn't contains partitions info.Is that a bug?

**Comment by hashhar** [comment]
which specific fields in the event output are you talking about?

In general engine has no concept of partition columns, specific connectors do.
